### PR TITLE
Cli Operator Unit Tests

### PR DESCRIPTION
**Pull request by Shaheer-Ahmd**
# Add CLI Operator Tests for Various Field Formats

This PR introduces a set of tests for the CLI operator functionality, covering various input field scenarios. The tests evaluate how the system handles different input values, such as single fields, multiple fields, empty fields, numbers as strings, and cases with a large number of fields. The tests ensure that the expected outputs are generated correctly in default, YAML, and JSON formats. (an effort towards #3835 )

The changes include the addition of two major test functions: `TestCliOperators` and `TestJsonArray`. Each test checks the correct output for different combinations of input values. This includes checking if the program formats single and multiple fields correctly in different serialization formats (default, YAML, JSON), and ensures proper handling of edge cases like empty values and large sets of fields.

## How to use

Reviewers can validate this PR by running the tests in the `pkg/operators/cli` package. To run the tests, use the following command:

```
go test -v ./pkg/operators/cli
```

Ensure that all tests pass and confirm that the outputs are correct for each input scenario.

## Testing done

I ran the tests for the CLI operator functionality using the following command:

```
go test -v ./pkg/operators/cli
```

All tests passed successfully. The output for each test case was checked against the expected results, and it correctly handled various scenarios including:

- Single and multiple fields with values.
- Empty values and fields.
- Handling of numbers as strings.
- Large number of fields.

Each output format (default, YAML, and JSON) was validated, ensuring the system's behavior aligns with the expectations defined in the test cases.

### CODE REVIEW

File: pkg/operators/cli/clioperator.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 The Inspektor Gadget authors
+// Copyright 2024-25 The Inspektor Gadget authors
```
Comment (burak-ok) [inline pkg/operators/cli/clioperator.go:1]: I think we use the full year everywhere else
```suggestion
// Copyright 2024-2025 The Inspektor Gadget authors
```

File: pkg/operators/cli/clioperator_test.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/datasource"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/datasource/formatters/json"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service/api"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCliOperators(t *testing.T) {
```
Comment (burak-ok) [inline pkg/operators/cli/clioperator_test.go:30]: Could you add another test function (can be a single test) with nested values?

**Review by burak-ok — COMMENTED** [review]
Looks good, thanks

**Review by burak-ok — APPROVED** [review]
